assets/armsnippets.jsonc

```diff
@@ -1108,10 +1108,10 @@
             "        \"enabledForDeployment\": true,",
             "        \"enabledForTemplateDeployment\": true,",
             "        \"enabledForDiskEncryption\": true,",
-            "        \"tenantId\": \"${2:tenantId}\",",
+            "        \"tenantId\": \"${2:00000000-0000-0000-0000-000000000000}\",",

```

Comment:
Did you decide as a general rule we wouldn't mention variables?

Reply:
I replaced tenantId with a dummy value that validates, however I guess it's moot since this value will need to be updated. I will revert.

I would love to use variable / parameters in the snippets if we could also create the corresponding template variables / parameters.

Reply:
Fixed